src/redis-benchmark.c

```diff
@@ -229,19 +229,13 @@ static long long ustime(void) {
     long long ust;
 
     gettimeofday(&tv, NULL);
-    ust = ((long)tv.tv_sec)*1000000;
+    ust = ((long long)tv.tv_sec)*1000000;
```

Comment:
This is a subtle overflow bug, I see other `ustime` in redis-cli redis-server has been fixed.
I'm wondering why redis tools implement some common functions repeatedly...

Reply:
i suppose it's like that because these functions are very simple and it was preferred to copy them rather than add dependency between the two programs.
maybe we'll do some additional cleanup some day, but at this point i'm ok to let it be.